themes/custom-blist/layouts/partials/homepage/social.html

```diff
@@ -12,13 +12,13 @@
       justify-center
     "
   >
-    <div class="max-w-xs text-center lg:text-left">
-      <div class="text-4xl font-extrabold mb-2">
-        {{ i18n "homeSocialTitle" }}
-      </div>
-      <p class="text-lg text-normal opacity-60">
-        {{ i18n "homeSocialDescription" }}
-      </p>
+    <div class="max-w-xs">
+      {{ partial "components/section-title"
+      (dict
+      "title" (i18n "homeSocialTitle")
+      "anchor" "socials"
+      "fontSize" "text-2xl" ) }}
```

Comment:
attention, après rebase la fontSize a changée pour cette section (homogénéisée avec les autres sections). On peut donc le supprimer des paramètres du composant